Makefile

```diff
@@ -225,6 +225,14 @@ migrate-api:  ## run alembic database migrations for the api service
 	@bin/alembic upgrade head
 .PHONY: migrate-api
 
+migrate-dashboard-db: ## create dashboard database
+	@echo "Creating dashboard service database…"
+	@$(COMPOSE_UP) --wait postgresql
+	@echo "Creating dashboard service database…"
+	@$(COMPOSE) exec postgresql bash -c 'psql "postgresql://$${POSTGRES_USER}:$${POSTGRES_PASSWORD}@$${QUALICHARGE_DB_HOST}:$${QUALICHARGE_DB_PORT}/postgres" -c "create database \"$${DASHBOARD_DB_NAME}\";"' || echo "Duly noted, skipping database creation."
+	@bin/manage migrate
+.PHONY: migrate-dashboard-db
```

Comment:
You need to separate database creation from migrations. The first is done once while the second can be called multiple times while working on the project.